core/storage/suggestion/gae_models.py

```diff
@@ -893,15 +896,19 @@ def get_in_review_question_suggestions_by_offset(
                     returned by the current query.
         """
 
+        filters = [
+            cls.status == STATUS_IN_REVIEW,
+            cls.suggestion_type == feconf.SUGGESTION_TYPE_ADD_QUESTION,
+        ]
+        if skill_ids:
```

Comment:
This is a bit weird logic, basically, when the list of IDs is empty you want to skip any checking of IDs? I would rather pass `None` as `skill_ids` in that case.

Reply:
Done. Thanks for reviewing.